examples/demo/src/routes/HomePage.tsx

```diff
@@ -1,10 +1,9 @@
 import algoliarecommend from '@algolia/recommend';
 import {
-  Recommend,
   TrendingFacets,
   TrendingItems,
   RecommendedForYou,
-} from '@algolia/recommend-react';
+} from '@algolia/recommend-react/dist/esm/experimental-personalization';
```

Comment:
```suggestion
} from '@algolia/recommend-react/experimental-personalization';
```

this would be possible with `exports` in package.json (but not sure if it's worth it of course)

Reply:
Good catch, i thought about adding some config for it, but for a beta feature where we will guide the initial clients during the setup, i don't think it is really needed.

Reply:
alternatively if you put it directly in a `experimental-personalization` folder instead of inside dist/esm, that should work too, but again, same considerations apply.